dashboard/src/main/webapp/src/pages/ApplicationPage.vue

```diff
@@ -55,7 +55,7 @@
       >
         <el-table-column prop="appId" label="AppId" min-width="180" sortable fixed />
         <el-table-column prop="userName" label="UserName" min-width="180" sortable />
-        <el-table-columnsortable
+        <el-table-column
           prop="registrationTime"
           label="Registration Time"
           min-width="180"
```

Comment:
The sortable is in L63

https://github.com/apache/incubator-uniffle/pull/1963/files#diff-f9bf9c7f3ab29a053cbc120fd3db975dce2e1f52b4e4295b90e08aaa840930c9R63

Reply:
> The sortable is in L63
> 
> https://github.com/apache/incubator-uniffle/pull/1963/files#diff-f9bf9c7f3ab29a053cbc120fd3db975dce2e1f52b4e4295b90e08aaa840930c9R63

OK. LGTM!